src/modules/popover/content.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { forwardRef } from 'react'
+
+import styles from './popover.css'
+
+import Card from 'elements/card'
+import { classNames } from 'utils'
+
+/**
+ * @visibleName Popover.Content
+ */
+const PopoverContent = forwardRef(
+  ({ children, className, ...passProps }, ref) => (
+    <Card
```

Comment:
The `Card` might be too heavy for the tooltips. It's better to create something else even if it looks similar.